sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/expressions/MathFunctionsSuite.scala

```diff
@@ -95,6 +94,19 @@ class MathFunctionsSuite extends SparkFunSuite with ExpressionEvalHelper {
     checkEvaluation(c(Literal(1.0), Literal.create(null, DoubleType)), null, create_row(null))
   }
 
+  test("conv") {
```

Comment:
conv("11abc", 10, 16)  it would return B. That's if there's any invalid digit in the number, the longest valid prefix would be converted.